scripts/install-windows-prereqs.ps1

```diff
@@ -16,6 +16,7 @@ Param(
     [string]$Clang7Hash = '672E4C420D6543A8A9F8EC5F1E5F283D88AC2155EF4C57232A399160A02BFF57',
     [string]$IntelPSWURL = 'http://registrationcenter-download.intel.com/akdlm/irc_nas/16607/Intel%20SGX%20PSW%20for%20Windows%20v2.7.101.2.exe',
     [string]$IntelPSWHash = 'AF669A4593411E9AABCE18838C91003866DDDEDAC5BEEC61DE160025008B0A19',
+    [string]$IntelPSWInfHash = '48e7c1e9-6de8-46ee-8ff9-46aa7b7ee5b9',
```

Comment:
I have a draft PR because of the new directory structure (https://github.com/openenclave/openenclave/pull/3249). This line wouldn't be needed anymore due to that change.